src/tests/functional/shared_test_classes/src/base/ov_subgraph.cpp

```diff
@@ -561,5 +550,107 @@ void SubgraphBaseTest::init_input_shapes(const std::vector<InputShape>& shapes)
         }
     }
 }
+
+void SubgraphBaseTest::compare_nodes(const std::shared_ptr<ov::Node>& node1, const std::shared_ptr<ov::Node>& node2, std::ostream& err_log) {
+    // compare inputs size, element_type and constant's values
+    if (node1->get_input_size() != node2->get_input_size()) {
+        err_log << "Number of inputs is different: " << to_str(node1->get_input_size()) << " for "
```

Comment:
What do you think about to replace `err_log` in the signature by throwing of exceptions?

Reply:
I use err_log, because I think that it is helpful to collect more information about all difference